runtime/devnet/src/config/assets.rs

```diff
@@ -12,6 +8,11 @@ use pallet_nfts::PalletFeatures;
 use parachains_common::{AssetIdForTrustBackedAssets, CollectionId, ItemId, Signature};
 use sp_runtime::traits::Verify;
 
+use crate::{
```

Comment:
Why are we leaving spaces between imports? It infers some cryptic separation that some maintainer needs to know about and adds no value. Far easier to group them into a single block and let cargo fmt handle consistency.

Reply:
External and local crates, rust best practice. This is how it is done in the pop cli code (maybe not the code you wrote). I'm fine with both but I would like to make a decision here so that it doesn't effect reviewing and writing code in the future of our repos.

Reply:
Source?

Id prefer to not have to worry about this, but https://rust-lang.github.io/rustfmt/?version=master&search=import#group_imports isn't stable. Formatting should be consistent but automated imo.